src/robotlibcore.py

```diff
@@ -142,6 +135,38 @@ def get_keyword_documentation(self, name):
             doc = '{}\n\n{}'.format(doc, tags) if doc else tags
         return doc
 
+    def get_keyword_types(self, keyword_name):
+        method = self.keywords.get(keyword_name)
+        if keyword_name == '__init__':
+            method = self.__init__
```

Comment:
1. Perhaps `if/else` would be a bit cleaner way to set `method` conditionally. 
2. Should this be done with all `get_keyword_xxx` methods? If yes, perhaps it would be a good idea to add a dedicated helper method like `def __get_keyword(self, name)` that does it. Notice the duple `_` prefix to make it certain extending libs don't accidentally override it.

Reply:
It feels also complicated for me. I did try to make look simpler.